tests/models/rl/unit/test_ppo.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+from unittest import TestCase
+
+import numpy as np
+import torch
+
+from pl_bolts.models.rl.ppo_model import PPO
+
+
+class TestPPOCategorical(TestCase):
+
+    def setUp(self) -> None:
+
+        parent_parser = argparse.ArgumentParser(add_help=False)
+        parent_parser = PPO.add_model_specific_args(parent_parser)
+        args_list = [
+            "--env",
+            "CartPole-v0",
+            "--batch_size",
+            "16",
+            "--gamma",
+            "0.99",
+        ]
+        self.hparams = parent_parser.parse_args(args_list)
```

Comment:
How about writing this like:
```suggestion
        cli_args = "--env CartPole-v0 --batch_size 16 --gamma 0.99".split()
        self.hparams = parent_parser.parse_args(cli_args)
```